{"total_count": 15, "incomplete_results": false, "items": [{"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/39", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/39/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/39/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/39/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/39", "id": 463476065, "node_id": "MDU6SXNzdWU0NjM0NzYwNjU=", "number": 39, "title": "pytest-remotedata is not compatible with dev versioning of pytest", "user": {"login": "bsipocz", "id": 6788290, "node_id": "MDQ6VXNlcjY3ODgyOTA=", "avatar_url": "https://avatars1.githubusercontent.com/u/6788290?v=4", "gravatar_id": "", "url": "https://api.github.com/users/bsipocz", "html_url": "https://github.com/bsipocz", "followers_url": "https://api.github.com/users/bsipocz/followers", "following_url": "https://api.github.com/users/bsipocz/following{/other_user}", "gists_url": "https://api.github.com/users/bsipocz/gists{/gist_id}", "starred_url": "https://api.github.com/users/bsipocz/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/bsipocz/subscriptions", "organizations_url": "https://api.github.com/users/bsipocz/orgs", "repos_url": "https://api.github.com/users/bsipocz/repos", "events_url": "https://api.github.com/users/bsipocz/events{/privacy}", "received_events_url": "https://api.github.com/users/bsipocz/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 698232331, "node_id": "MDU6TGFiZWw2OTgyMzIzMzE=", "url": "https://api.github.com/repos/astropy/pytest-remotedata/labels/bug", "name": "bug", "color": "ee0701", "default": true, "description": null}], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 0, "created_at": "2019-07-02T23:20:00Z", "updated_at": "2019-07-03T20:38:58Z", "closed_at": "2019-07-03T20:38:58Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "The pytest version number for the developer branch is not compatible with `StrictVersion`, and thus I run into tons of errors like the one below when tried to use pytest dev with astropy core. \r\n\r\nI don't see a reason of why `StrictVersion` is preferred instead of `LooseVersion`. If there is something I miss @drdavella  please let me know.\r\n\r\n```\r\n____________________________________________________ ERROR at setup of test_numpy_types[c5] _____________________________________________________\r\n\r\nitem = <Function test_numpy_types[c5]>\r\n\r\n    def pytest_runtest_setup(item):\r\n    \r\n>       if StrictVersion(pytest.__version__) < StrictVersion(\"3.6\"):\r\n\r\n/usr/local/lib/python3.7/site-packages/pytest_remotedata/plugin.py:65: \r\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\r\n/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/version.py:40: in __init__\r\n    self.parse(vstring)\r\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\r\n\r\nself = <[AttributeError(\"'StrictVersion' object has no attribute 'version'\") raised in repr()] StrictVersion object at 0x13a052c88>\r\nvstring = '5.0.1.dev15+g5d8da88e9.d20190702'\r\n\r\n    def parse (self, vstring):\r\n        match = self.version_re.match(vstring)\r\n        if not match:\r\n>           raise ValueError(\"invalid version number '%s'\" % vstring)\r\nE           ValueError: invalid version number '5.0.1.dev15+g5d8da88e9.d20190702'\r\n\r\n/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/version.py:137: ValueError\r\n```", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/37", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/37/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/37/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/37/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/37", "id": 409002231, "node_id": "MDU6SXNzdWU0MDkwMDIyMzE=", "number": 37, "title": "Pytest >= 4.1 throws AttributeError on all tests for pytest-remotedata", "user": {"login": "rawrgulmuffins", "id": 1521575, "node_id": "MDQ6VXNlcjE1MjE1NzU=", "avatar_url": "https://avatars0.githubusercontent.com/u/1521575?v=4", "gravatar_id": "", "url": "https://api.github.com/users/rawrgulmuffins", "html_url": "https://github.com/rawrgulmuffins", "followers_url": "https://api.github.com/users/rawrgulmuffins/followers", "following_url": "https://api.github.com/users/rawrgulmuffins/following{/other_user}", "gists_url": "https://api.github.com/users/rawrgulmuffins/gists{/gist_id}", "starred_url": "https://api.github.com/users/rawrgulmuffins/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/rawrgulmuffins/subscriptions", "organizations_url": "https://api.github.com/users/rawrgulmuffins/orgs", "repos_url": "https://api.github.com/users/rawrgulmuffins/repos", "events_url": "https://api.github.com/users/rawrgulmuffins/events{/privacy}", "received_events_url": "https://api.github.com/users/rawrgulmuffins/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 4, "created_at": "2019-02-11T22:16:26Z", "updated_at": "2019-07-03T20:42:01Z", "closed_at": "2019-07-03T20:42:00Z", "author_association": "NONE", "active_lock_reason": null, "body": "[Related thread from pytest-cov.](https://github.com/pytest-dev/pytest-cov/issues/252#issuecomment-462246411)\r\n\r\nSpecific error raised.\r\n\r\n```\r\npython -m pytest\r\n======================================== test session starts =========================================\r\nplatform darwin -- Python 3.6.5, pytest-4.2.0, py-1.5.3, pluggy-0.8.1\r\nbenchmark: 3.2.2 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)\r\nrootdir: /Users/alexl/git/example, inifile: setup.cfg\r\nplugins: remotedata-0.2.1, openfiles-0.3.0, doctestplus-0.1.3, cov-2.6.1, benchmark-3.2.2, arraydiff-0.2\r\ncollected 1 item\r\n\r\ntests/test_example.py E                                                                        [100%]\r\n\r\n=============================================== ERRORS ===============================================\r\n___________________________ ERROR at setup of test_command_line_interface ____________________________\r\n\r\nitem = <Function test_command_line_interface>\r\n\r\n    def pytest_runtest_setup(item):\r\n\r\n>       remote_data = item.get_marker('remote_data')\r\nE       AttributeError: 'Function' object has no attribute 'get_marker'\r\n\r\n/usr/local/anaconda3/lib/python3.6/site-packages/pytest_remotedata/plugin.py:59: AttributeError\r\n====================================== 1 error in 0.11 seconds =======================================\r\nmake: *** [test] Error 1\r\n```", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/35", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/35/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/35/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/35/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/35", "id": 375030629, "node_id": "MDU6SXNzdWUzNzUwMzA2Mjk=", "number": 35, "title": "Deprecation warnings with pytest 3.8", "user": {"login": "astrofrog", "id": 314716, "node_id": "MDQ6VXNlcjMxNDcxNg==", "avatar_url": "https://avatars2.githubusercontent.com/u/314716?v=4", "gravatar_id": "", "url": "https://api.github.com/users/astrofrog", "html_url": "https://github.com/astrofrog", "followers_url": "https://api.github.com/users/astrofrog/followers", "following_url": "https://api.github.com/users/astrofrog/following{/other_user}", "gists_url": "https://api.github.com/users/astrofrog/gists{/gist_id}", "starred_url": "https://api.github.com/users/astrofrog/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/astrofrog/subscriptions", "organizations_url": "https://api.github.com/users/astrofrog/orgs", "repos_url": "https://api.github.com/users/astrofrog/repos", "events_url": "https://api.github.com/users/astrofrog/events{/privacy}", "received_events_url": "https://api.github.com/users/astrofrog/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 3, "created_at": "2018-10-29T14:02:27Z", "updated_at": "2019-07-03T20:42:50Z", "closed_at": "2019-07-03T20:42:50Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "I'm seeing the following deprecation warnings when using pytest-remotedata with pytest 3.8:\r\n\r\n```\r\n/Users/tom/miniconda3/envs/dev/lib/python3.7/site-packages/pytest_remotedata/plugin.py:73: RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.\r\nPlease use node.get_closest_marker(name) or node.iter_markers(name).\r\nDocs: https://docs.pytest.org/en/latest/mark.html#updating-code\r\n  if len(remote_data.args) > 0:\r\n/Users/tom/miniconda3/envs/dev/lib/python3.7/site-packages/pytest_remotedata/plugin.py:76: RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.\r\nPlease use node.get_closest_marker(name) or node.iter_markers(name).\r\nDocs: https://docs.pytest.org/en/latest/mark.html#updating-code\r\n  source = remote_data.kwargs.get('source', 'any')\r\n```", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/31", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/31/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/31/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/31/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/31", "id": 328629019, "node_id": "MDU6SXNzdWUzMjg2MjkwMTk=", "number": 31, "title": "RemovedInPytest4Warning: MarkInfo objects are deprecated", "user": {"login": "pllim", "id": 2090236, "node_id": "MDQ6VXNlcjIwOTAyMzY=", "avatar_url": "https://avatars2.githubusercontent.com/u/2090236?v=4", "gravatar_id": "", "url": "https://api.github.com/users/pllim", "html_url": "https://github.com/pllim", "followers_url": "https://api.github.com/users/pllim/followers", "following_url": "https://api.github.com/users/pllim/following{/other_user}", "gists_url": "https://api.github.com/users/pllim/gists{/gist_id}", "starred_url": "https://api.github.com/users/pllim/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/pllim/subscriptions", "organizations_url": "https://api.github.com/users/pllim/orgs", "repos_url": "https://api.github.com/users/pllim/repos", "events_url": "https://api.github.com/users/pllim/events{/privacy}", "received_events_url": "https://api.github.com/users/pllim/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 0, "created_at": "2018-06-01T19:07:43Z", "updated_at": "2018-10-18T13:57:50Z", "closed_at": "2018-10-18T13:57:50Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "This warning is currently ignored as per astropy/astropy#7490 but still should be addressed at some point?\r\n\r\n```\r\n    def pytest_runtest_setup(item):\r\n    \r\n        remote_data = item.get_marker('remote_data')\r\n        internet_off = item.get_marker('internet_off')\r\n    \r\n        remote_data_config = item.config.getvalue(\"remote_data\")\r\n    \r\n        if remote_data is not None and internet_off is not None:\r\n            raise ValueError(\"remote_data and internet_off are not compatible\")\r\n    \r\n        if remote_data is not None:\r\n>           source = remote_data.kwargs.get('source', 'any')\r\nE           _pytest.deprecated.RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.\r\nE           Please use node.get_closest_marker(name) or node.iter_markers(name).\r\nE           Docs: https://docs.pytest.org/en/latest/mark.html#updating-code\r\n/home/travis/miniconda/envs/test/lib/python3.6/site-packages/pytest_remotedata/plugin.py:68: RemovedInPytest4Warning\r\n```\r\n\r\nhttps://github.com/astropy/pytest-remotedata/blob/v0.3.0/pytest_remotedata/plugin.py#L76", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/26", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/26/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/26/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/26/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/26", "id": 325844788, "node_id": "MDU6SXNzdWUzMjU4NDQ3ODg=", "number": 26, "title": "--remote-data=astropy does not work properly?", "user": {"login": "pllim", "id": 2090236, "node_id": "MDQ6VXNlcjIwOTAyMzY=", "avatar_url": "https://avatars2.githubusercontent.com/u/2090236?v=4", "gravatar_id": "", "url": "https://api.github.com/users/pllim", "html_url": "https://github.com/pllim", "followers_url": "https://api.github.com/users/pllim/followers", "following_url": "https://api.github.com/users/pllim/following{/other_user}", "gists_url": "https://api.github.com/users/pllim/gists{/gist_id}", "starred_url": "https://api.github.com/users/pllim/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/pllim/subscriptions", "organizations_url": "https://api.github.com/users/pllim/orgs", "repos_url": "https://api.github.com/users/pllim/repos", "events_url": "https://api.github.com/users/pllim/events{/privacy}", "received_events_url": "https://api.github.com/users/pllim/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 5, "created_at": "2018-05-23T19:35:41Z", "updated_at": "2018-05-29T17:55:15Z", "closed_at": "2018-05-29T17:55:15Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "I am testing something in Astropy v2.0.x branch in Python 2 environment and I noticed that `--remote-data=astropy` and `--remote-data=none` skip the same tests even when I know there are some tests marked as `@remote_data('astropy')` in the test file. I was expecting that `--remote-data=astropy` would skip *less* tests (i.e., run more tests).\r\n\r\n```\r\n$ python setup.py test -t astropy/utils/tests/test_data.py --remote-data=astropy\r\n============================= test session starts ==============================\r\nplatform linux2 -- Python 2.7.14, pytest-3.5.0, py-1.5.3, pluggy-0.6.0\r\n\r\nRunning tests with Astropy version 2.0.7.dev19589.\r\nRunning tests in astropy/utils/tests/test_data.py.\r\n\r\nDate: 2018-05-23T15:30:08\r\n\r\nPlatform: Linux-3.10.0-862.el7.x86_64-x86_64-with-redhat-7.5-Maipo\r\n\r\nExecutable: .../anaconda/envs/iraf27/bin/python\r\n\r\nFull Python Version: \r\n2.7.14 |Anaconda, Inc.| (default, Mar 27 2018, 17:29:31) \r\n[GCC 7.2.0]\r\n\r\nencodings: sys: ascii, locale: UTF-8, filesystem: UTF-8, unicode bits: 20\r\nbyteorder: little\r\nfloat info: dig: 15, mant_dig: 15\r\n\r\nNumpy: 1.13.3\r\nScipy: 1.0.1\r\nMatplotlib: 2.2.2\r\nh5py: 2.7.1\r\nPandas: 0.22.0\r\nCython: 0.28.1\r\nUsing Astropy options: remote_data: astropy.\r\n\r\nrootdir: /tmp/astropy-test-2PR7lr/lib/python2.7/site-packages, inifile: setup.cfg\r\nplugins: remotedata-0.2.1, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2\r\ncollected 58 items                                                             \r\n\r\nastropy/utils/tests/test_data.py .sssssss..........s...x.s.s.s.sssssss.. [ 50%]\r\n........s...x.s.s.s\r\n<snipped>\r\n========= 34 passed, 22 skipped, 2 xfailed, 2 warnings in 0.38 seconds =========\r\n```\r\n\r\n```\r\n$ python setup.py test -t astropy/utils/tests/test_data.py\r\n<snipped>\r\n========= 34 passed, 22 skipped, 2 xfailed, 2 warnings in 0.39 seconds =========\r\n```\r\n\r\nBut when I run with `--remote-data` (any), they are not skipped anymore.\r\n\r\nHere is an example test function in the given test file:\r\n```python\r\n@remote_data('astropy')\r\ndef test_download_parallel():\r\n    ...\r\n```", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/23", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/23/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/23/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/23/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/23", "id": 316288371, "node_id": "MDU6SXNzdWUzMTYyODgzNzE=", "number": 23, "title": "Issue installing astropy via conda", "user": {"login": "lbdreyer", "id": 8101163, "node_id": "MDQ6VXNlcjgxMDExNjM=", "avatar_url": "https://avatars3.githubusercontent.com/u/8101163?v=4", "gravatar_id": "", "url": "https://api.github.com/users/lbdreyer", "html_url": "https://github.com/lbdreyer", "followers_url": "https://api.github.com/users/lbdreyer/followers", "following_url": "https://api.github.com/users/lbdreyer/following{/other_user}", "gists_url": "https://api.github.com/users/lbdreyer/gists{/gist_id}", "starred_url": "https://api.github.com/users/lbdreyer/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/lbdreyer/subscriptions", "organizations_url": "https://api.github.com/users/lbdreyer/orgs", "repos_url": "https://api.github.com/users/lbdreyer/repos", "events_url": "https://api.github.com/users/lbdreyer/events{/privacy}", "received_events_url": "https://api.github.com/users/lbdreyer/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 5, "created_at": "2018-04-20T14:23:45Z", "updated_at": "2018-04-20T19:44:07Z", "closed_at": "2018-04-20T17:22:39Z", "author_association": "NONE", "active_lock_reason": null, "body": "I tried installing `astropy` with miniconda and noticed there was an issue with installing `pytest-remotedata` and `pytest-openfiles`.\r\n\r\nThe anaconda recipe for `astropy 3.0.1` sets `pytest-astropy` as a runtime dependency, and `pytest-astropy` sets `pytest-remotedata` and `pytest-openfiles` as runtime dependencies. This means that when I try to install `astropy` into a conda environment, `pytest-remotedata` and `pytest-openfiles` (and `pytest-astropy`) also get installed, but there is an issue with installing both into a single environment due to their directory structures. \r\n\r\nBoth `pytest-remotedata` and `pytest-openfiles` have a top level `tests` directory which get installed into the `lib/python3.6/site-packages/tests` directory of the environment. So whichever package get installed second is trying overwrite whichever gets installed first. and this is causing problems.\r\n\r\nIs `pytest-astropy` a runtime dependency of `astropy`? Or should the anaconda recipe for `astropy`  be updated?\r\n\r\n\r\nBelow is the output I get when I try to install `astropy` with conda. See the `ClobberError` at the end for the error relating to `pytest-remotedata` and `pytest-openfiles`\r\n```\r\n$ conda create -n test_astropy_pytest astropy\r\nSolving environment: done\r\n\r\n## Package Plan ##\r\n\r\n  environment location: .../miniconda3/envs/test_astropy_pytest\r\n\r\n  added / updated specs: \r\n    - astropy\r\n\r\n\r\nThe following packages will be downloaded:\r\n\r\n    package                    |            build\r\n    ---------------------------|-----------------\r\n    psutil-5.4.5               |   py36h14c3975_0         303 KB  defaults\r\n    pluggy-0.6.0               |   py36hb689045_0          23 KB  defaults\r\n    mkl-2018.0.2               |                1       205.2 MB  defaults\r\n    pytest-openfiles-0.2.0     |           py36_0           9 KB  defaults\r\n    pytest-remotedata-0.2.0    |           py36_0          16 KB  defaults\r\n    attrs-17.4.0               |           py36_0          41 KB  defaults\r\n    pytest-arraydiff-0.2       |           py36_0          14 KB  defaults\r\n    libgfortran-ng-7.2.0       |       hdf63c60_3         1.2 MB  defaults\r\n    intel-openmp-2018.0.0      |                8         620 KB  defaults\r\n    setuptools-39.0.1          |           py36_0         549 KB  defaults\r\n    astropy-3.0.1              |   py36h3010b51_1         6.8 MB  defaults\r\n    pytest-3.5.0               |           py36_0         293 KB  defaults\r\n    numpy-1.14.2               |   py36hdbf6ddf_1         4.1 MB  defaults\r\n    ------------------------------------------------------------\r\n                                           Total:       219.2 MB\r\n\r\nThe following NEW packages will be INSTALLED:\r\n\r\n    astropy:            3.0.1-py36h3010b51_1  defaults\r\n    attrs:              17.4.0-py36_0         defaults\r\n    ca-certificates:    2018.03.07-0          defaults\r\n    certifi:            2018.4.16-py36_0      defaults\r\n    intel-openmp:       2018.0.0-8            defaults\r\n    libedit:            3.1-heed3624_0        defaults\r\n    libffi:             3.2.1-hd88cf55_4      defaults\r\n    libgcc-ng:          7.2.0-hdf63c60_3      defaults\r\n    libgfortran-ng:     7.2.0-hdf63c60_3      defaults\r\n    libstdcxx-ng:       7.2.0-hdf63c60_3      defaults\r\n    mkl:                2018.0.2-1            defaults\r\n    mkl_fft:            1.0.1-py36h3010b51_0  defaults\r\n    mkl_random:         1.0.1-py36h629b387_0  defaults\r\n    more-itertools:     4.1.0-py36_0          defaults\r\n    ncurses:            6.0-h9df7e31_2        defaults\r\n    numpy:              1.14.2-py36hdbf6ddf_1 defaults\r\n    openssl:            1.0.2o-h20670df_0     defaults\r\n    pluggy:             0.6.0-py36hb689045_0  defaults\r\n    psutil:             5.4.5-py36h14c3975_0  defaults\r\n    py:                 1.5.3-py36_0          defaults\r\n    pytest:             3.5.0-py36_0          defaults\r\n    pytest-arraydiff:   0.2-py36_0            defaults\r\n    pytest-astropy:     0.2.1-py36_0          defaults\r\n    pytest-doctestplus: 0.1.2-py36_0          defaults\r\n    pytest-openfiles:   0.2.0-py36_0          defaults\r\n    pytest-remotedata:  0.2.0-py36_0          defaults\r\n    python:             3.6.5-hc3d631a_0      defaults\r\n    readline:           7.0-ha6073c6_4        defaults\r\n    setuptools:         39.0.1-py36_0         defaults\r\n    six:                1.11.0-py36h372c433_1 defaults\r\n    sqlite:             3.23.1-he433501_0     defaults\r\n    tk:                 8.6.7-hc745277_3      defaults\r\n    xz:                 5.2.3-h55aa19d_2      defaults\r\n    zlib:               1.2.11-ha838bed_2     defaults\r\n\r\nProceed ([y]/n)? y\r\n\r\n\r\nDownloading and Extracting Packages\r\npsutil 5.4.5########################################################################################################################################## | 100% \r\npluggy 0.6.0########################################################################################################################################## | 100% \r\nmkl 2018.0.2########################################################################################################################################## | 100% \r\npytest-openfiles 0.2.0################################################################################################################################ | 100% \r\npytest-remotedata 0.2.0############################################################################################################################### | 100% \r\nattrs 17.4.0########################################################################################################################################## | 100% \r\npytest-arraydiff 0.2################################################################################################################################## | 100% \r\nlibgfortran-ng 7.2.0################################################################################################################################## | 100% \r\nintel-openmp 2018.0.0################################################################################################################################# | 100% \r\nsetuptools 39.0.1##################################################################################################################################### | 100% \r\nastropy 3.0.1######################################################################################################################################### | 100% \r\npytest 3.5.0########################################################################################################################################## | 100% \r\nnumpy 1.14.2########################################################################################################################################## | 100% \r\nPreparing transaction: done\r\nVerifying transaction: failed\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/__pycache__/_assertionold.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/__pycache__/code.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/__pycache__/source.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/_assertionold.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/code.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_code/source.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_io/capture.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_io/terminalwriter.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/__pycache__/common.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/__pycache__/local.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/__pycache__/svnurl.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/__pycache__/svnwc.cpython-36.pyc'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/common.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/local.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/svnurl.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_path/svnwc.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nCondaVerificationError: The package for py located at .../miniconda3/pkgs/py-1.5.3-py36_0\r\nappears to be corrupted. The path 'lib/python3.6/site-packages/py/_vendored_packages/__init__.py'\r\nspecified in the package manifest cannot be found.\r\n\r\nClobberError: This transaction has incompatible packages due to a shared path.\r\n  packages: defaults::pytest-openfiles-0.2.0-py36_0, defaults::pytest-remotedata-0.2.0-py36_0\r\n  path: 'lib/python3.6/site-packages/tests/__init__.py'\r\n\r\n\r\nClobberError: This transaction has incompatible packages due to a shared path.\r\n  packages: defaults::pytest-openfiles-0.2.0-py36_0, defaults::pytest-remotedata-0.2.0-py36_0\r\n  path: 'lib/python3.6/site-packages/tests/__pycache__/__init__.cpython-36.pyc'\r\n```\r\n", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/19", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/19/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/19/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/19/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/19", "id": 272873945, "node_id": "MDU6SXNzdWUyNzI4NzM5NDU=", "number": 19, "title": "New release?", "user": {"login": "astrofrog", "id": 314716, "node_id": "MDQ6VXNlcjMxNDcxNg==", "avatar_url": "https://avatars2.githubusercontent.com/u/314716?v=4", "gravatar_id": "", "url": "https://api.github.com/users/astrofrog", "html_url": "https://github.com/astrofrog", "followers_url": "https://api.github.com/users/astrofrog/followers", "following_url": "https://api.github.com/users/astrofrog/following{/other_user}", "gists_url": "https://api.github.com/users/astrofrog/gists{/gist_id}", "starred_url": "https://api.github.com/users/astrofrog/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/astrofrog/subscriptions", "organizations_url": "https://api.github.com/users/astrofrog/orgs", "repos_url": "https://api.github.com/users/astrofrog/repos", "events_url": "https://api.github.com/users/astrofrog/events{/privacy}", "received_events_url": "https://api.github.com/users/astrofrog/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 2, "created_at": "2017-11-10T09:41:19Z", "updated_at": "2017-11-15T15:48:39Z", "closed_at": "2017-11-15T15:48:39Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "@drdavella - could you do a new release? The latest stable version doesn't include the fixes to the mirror infrastructure, so builds are failing on astropy master again due to this.", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/18", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/18/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/18/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/18/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/18", "id": 272721284, "node_id": "MDU6SXNzdWUyNzI3MjEyODQ=", "number": 18, "title": "Plugin blocks internet access for all tests not using decorator", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 698232331, "node_id": "MDU6TGFiZWw2OTgyMzIzMzE=", "url": "https://api.github.com/repos/astropy/pytest-remotedata/labels/bug", "name": "bug", "color": "ee0701", "default": true, "description": null}, {"id": 745832217, "node_id": "MDU6TGFiZWw3NDU4MzIyMTc=", "url": "https://api.github.com/repos/astropy/pytest-remotedata/labels/high-priority", "name": "high-priority", "color": "db482e", "default": false, "description": null}], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 0, "created_at": "2017-11-09T20:56:38Z", "updated_at": "2017-11-10T15:14:11Z", "closed_at": "2017-11-10T15:14:11Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "Currently, if this plugin is installed it will block internet access to **all** tests that are not marked with ``@pytest.mark.remote_data``. This is the desired behavior for `astropy` and some affiliated packages. However, just having this plugin installed will affect other packages that don't care about the ``remote_data`` plugin.\r\n\r\nIt seems like the best solution is to have a configuration option to enable the strict behavior. Packages that wish to enable this behavior will add a field to their `setup.cfg` file. Packages that don't care (and therefore don't have this option in their `setup.cfg` file) won't be affected.", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/12", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/12/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/12/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/12/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/12", "id": 265343323, "node_id": "MDU6SXNzdWUyNjUzNDMzMjM=", "number": 12, "title": "Port changes to data file mirror in astropy", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 0, "created_at": "2017-10-13T16:14:55Z", "updated_at": "2017-10-16T13:11:43Z", "closed_at": "2017-10-16T13:11:43Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "The changes are in https://github.com/astropy/astropy/pull/6724.", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/11", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/11/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/11/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/11/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/11", "id": 264317031, "node_id": "MDU6SXNzdWUyNjQzMTcwMzE=", "number": 11, "title": "Can/should testing dependency on astropy be removed?", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 1, "created_at": "2017-10-10T17:43:43Z", "updated_at": "2017-10-16T19:13:19Z", "closed_at": "2017-10-16T19:13:19Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/8", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/8/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/8/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/8/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/8", "id": 262924490, "node_id": "MDU6SXNzdWUyNjI5MjQ0OTA=", "number": 8, "title": "Remove \"addons\" section and consolidate travis test matrix", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 1, "created_at": "2017-10-04T20:44:54Z", "updated_at": "2017-10-09T14:53:49Z", "closed_at": "2017-10-09T14:53:49Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/7", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/7/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/7/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/7/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/7", "id": 262846266, "node_id": "MDU6SXNzdWUyNjI4NDYyNjY=", "number": 7, "title": "Do not accept invalid arguments to --remote-data", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": {"url": "https://api.github.com/repos/astropy/pytest-remotedata/milestones/1", "html_url": "https://github.com/astropy/pytest-remotedata/milestone/1", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/milestones/1/labels", "id": 2810688, "node_id": "MDk6TWlsZXN0b25lMjgxMDY4OA==", "number": 1, "title": "Beta Release", "description": "", "creator": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "open_issues": 0, "closed_issues": 5, "state": "open", "created_at": "2017-10-03T14:46:49Z", "updated_at": "2017-10-16T19:13:20Z", "due_on": "2017-10-27T07:00:00Z", "closed_at": null}, "comments": 0, "created_at": "2017-10-04T16:13:08Z", "updated_at": "2017-10-16T13:11:22Z", "closed_at": "2017-10-16T13:11:22Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "For example, passing `--remote-data=foo` should cause an error as early as possible.", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/5", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/5/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/5/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/5/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/5", "id": 261949584, "node_id": "MDU6SXNzdWUyNjE5NDk1ODQ=", "number": 5, "title": "Remotedata rst handler", "user": {"login": "MSeifert04", "id": 14200878, "node_id": "MDQ6VXNlcjE0MjAwODc4", "avatar_url": "https://avatars2.githubusercontent.com/u/14200878?v=4", "gravatar_id": "", "url": "https://api.github.com/users/MSeifert04", "html_url": "https://github.com/MSeifert04", "followers_url": "https://api.github.com/users/MSeifert04/followers", "following_url": "https://api.github.com/users/MSeifert04/following{/other_user}", "gists_url": "https://api.github.com/users/MSeifert04/gists{/gist_id}", "starred_url": "https://api.github.com/users/MSeifert04/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/MSeifert04/subscriptions", "organizations_url": "https://api.github.com/users/MSeifert04/orgs", "repos_url": "https://api.github.com/users/MSeifert04/repos", "events_url": "https://api.github.com/users/MSeifert04/events{/privacy}", "received_events_url": "https://api.github.com/users/MSeifert04/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 11, "created_at": "2017-10-01T21:53:32Z", "updated_at": "2017-10-02T16:22:54Z", "closed_at": "2017-10-02T16:20:24Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "Not sure if this would be a good fit for this package but is it possible to make a code-block-like `skip-remote` directive that would work like the decorator for the pytest functions, i.e. only runs the doctest if `--remote-data` has been specified?", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/3", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/3/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/3/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/3/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/3", "id": 260405541, "node_id": "MDU6SXNzdWUyNjA0MDU1NDE=", "number": 3, "title": "Add a decorator for tests that should run only when no connection is available", "user": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 1, "created_at": "2017-09-25T20:32:31Z", "updated_at": "2017-09-26T13:11:11Z", "closed_at": "2017-09-26T13:11:11Z", "author_association": "CONTRIBUTOR", "active_lock_reason": null, "body": "This will allow for code paths to be tested that expect remote data access to be disabled. Previously some tests in astropy tried to provide for this by accessing the `INTERNET_OFF` global variable, but this is no longer directly available to test code since it lives inside of this plugin.", "performed_via_github_app": null, "score": 1.0}, {"url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/2", "repository_url": "https://api.github.com/repos/astropy/pytest-remotedata", "labels_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/2/labels{/name}", "comments_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/2/comments", "events_url": "https://api.github.com/repos/astropy/pytest-remotedata/issues/2/events", "html_url": "https://github.com/astropy/pytest-remotedata/issues/2", "id": 259311026, "node_id": "MDU6SXNzdWUyNTkzMTEwMjY=", "number": 2, "title": "Don't have src directory", "user": {"login": "astrofrog", "id": 314716, "node_id": "MDQ6VXNlcjMxNDcxNg==", "avatar_url": "https://avatars2.githubusercontent.com/u/314716?v=4", "gravatar_id": "", "url": "https://api.github.com/users/astrofrog", "html_url": "https://github.com/astrofrog", "followers_url": "https://api.github.com/users/astrofrog/followers", "following_url": "https://api.github.com/users/astrofrog/following{/other_user}", "gists_url": "https://api.github.com/users/astrofrog/gists{/gist_id}", "starred_url": "https://api.github.com/users/astrofrog/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/astrofrog/subscriptions", "organizations_url": "https://api.github.com/users/astrofrog/orgs", "repos_url": "https://api.github.com/users/astrofrog/repos", "events_url": "https://api.github.com/users/astrofrog/events{/privacy}", "received_events_url": "https://api.github.com/users/astrofrog/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": {"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "drdavella", "id": 2458487, "node_id": "MDQ6VXNlcjI0NTg0ODc=", "avatar_url": "https://avatars0.githubusercontent.com/u/2458487?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drdavella", "html_url": "https://github.com/drdavella", "followers_url": "https://api.github.com/users/drdavella/followers", "following_url": "https://api.github.com/users/drdavella/following{/other_user}", "gists_url": "https://api.github.com/users/drdavella/gists{/gist_id}", "starred_url": "https://api.github.com/users/drdavella/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drdavella/subscriptions", "organizations_url": "https://api.github.com/users/drdavella/orgs", "repos_url": "https://api.github.com/users/drdavella/repos", "events_url": "https://api.github.com/users/drdavella/events{/privacy}", "received_events_url": "https://api.github.com/users/drdavella/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 1, "created_at": "2017-09-20T21:42:18Z", "updated_at": "2017-09-22T17:19:57Z", "closed_at": "2017-09-22T17:19:57Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "For pytest plugins, it's not common to put the source inside the ``src`` directory. So the ``pytest_remotedata`` directory should be moved up to the top level.", "performed_via_github_app": null, "score": 1.0}]}